source/_integrations/homekit.markdown

```diff
@@ -489,6 +542,14 @@ Media Player entities with `device_class: tv` will show up as Television accesso
 
 The volume and play/pause controls will show up on the Remote app or Control Center. If your TV supports volume control through Home Assistant, you will be able to control the volume using the side volume buttons on the device while having the remote selected on screen.
 
+#### Camera video is not streaming
+
+Ensure you have a recent version of FFmpeg installed. Verify that your stream is directly playable with `ffplay <stream_source>`. If you have changed your camera's entity configuration, you may need to [reset the accessory](#resetting-accessories).
+
+#### Camera audio is not streaming
+
+Make sure `support_audio` is `True` in the camera's entity configuration. FFmpeg *must* be compiled with `--with-fdk-aac` for audio support. Check with `ffmpeg -encoders`, ensuring `libfdk_aac` is in the list.
```

Comment:
This isn't checkable/verifiable for Home Assistant users, or users using Home Assistant Core in Docker.

Reply:
I feel like the integration should check for the encoder on startup, warn, and disable audio. So this check wouldn't really be necessary. Now, how they would build/install a version of ffmpeg with the encoder... 🤷‍♀

Reply:
Well, I've checked, it isn't compiled in:

`--prefix=/usr --enable-avresample --enable-avfilter --enable-gnutls --enable-gpl --enable-libass --enable-libmp3lame --enable-libvorbis --enable-libvpx --enable-libxvid --enable-libx264 --enable-libx265 --enable-libtheora --enable-libv4l2 --enable-postproc --enable-pic --enable-pthreads --enable-shared --enable-libxcb --disable-stripping --disable-static --disable-librtmp --enable-vaapi --enable-vdpau --enable-libopus --disable-debug`

`libfdk_aac` isn't in the list of encoders either.

Reply:
Resolved by defaulting to libopus which is enabled by default.